org-code-javabuilder/lib/src/test/java/org/code/javabuilder/AWSInputAdapterTest.java

```diff
@@ -0,0 +1,63 @@
+package org.code.javabuilder;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.Message;
+import com.amazonaws.services.sqs.model.ReceiveMessageRequest;
+import com.amazonaws.services.sqs.model.ReceiveMessageResult;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class AWSInputAdapterTest {
+  private AWSInputAdapter inputAdapter;
+
+  private AmazonSQS sqsMock;
+
+  @BeforeEach
+  public void setUp() {
+    Message firstMessage = mock(Message.class);
+    Message secondMessage = mock(Message.class);
+    when(firstMessage.getBody()).thenReturn("hello");
+    when(secondMessage.getBody()).thenReturn("world");
+    List<Message> messages = new ArrayList<>();
+    messages.add(firstMessage);
+    messages.add(secondMessage);
+
+    ReceiveMessageResult result = mock(ReceiveMessageResult.class);
+    when(result.getMessages()).thenReturn(messages);
+    sqsMock = mock(AmazonSQS.class);
+    when(sqsMock.receiveMessage(any(ReceiveMessageRequest.class))).thenReturn(result);
```

Comment:
Some of this setup looks test-specific.  I recommend moving anything that is test-specific out to the individual test themselves, adding helper functions to reduce code duplication if needed.

Reply:
Thanks! That cleaned things up a bunch!